Azayaka/Preferences.swift

```diff
@@ -43,6 +45,9 @@ struct Preferences: View {
                 Toggle(isOn: $hideSelf) {
                     Text("Exclude Azayaka itself")
                 }.toggleStyle(CheckboxToggleStyle())
+                Toggle(isOn: $frontApp) {
+                    Text("Include only front app")
```

Comment:
I think "Only list focused app's windows" could be a clearer description, if it is not too long - what do you think?

Reply:
Sure - I think the decision is yours, and don't have better options.

I had trouble getting wording to not expand the menu overly wide.  I adopted `Include` because you already had `Exclude`.  

Variants considered:

- [List | Include] {only} front [App | application | program]
- Focus on front App

Characterizing the target is hard:
- "Focused" is a term of art
- "Selected" is more common for users but not really apt
- "Front" might not understand - which is "front"?

Reply:
I see - makes sense to me! Might be worth exploring some other UI at some point if I have the time, your ideas sound interesting, so thanks to you too :)

"Focus" is usually used for apps/windows as far as I know so I think that'll be best.